### PR TITLE
test/helpers: remove unused functions and consts

### PR DESCRIPTION
**Pull request by tklauser**
These are unused since commit 53736573fc59 ("test: Remove deprecated
Nightly tests").

### CODE REVIEW

**Comment by tklauser** [comment]
test-me-please

**Comment by tklauser** [comment]
test-1.19-4.19

**Comment by tklauser** [comment]
test-1.20-4.9

**Comment by tklauser** [comment]
test-1.19-4.19

**Comment by tklauser** [comment]
test-1.20-4.9

File: test/helpers/cons.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ import (
 
 	k8sConst "github.com/cilium/cilium/pkg/k8s/apis/cilium.io"
 	"github.com/cilium/cilium/pkg/versioncheck"
-	"github.com/cilium/cilium/test/ginkgo-ext"
+	ginkgoext "github.com/cilium/cilium/test/ginkgo-ext"
```
Comment (kkourt) [inline test/helpers/cons.go:25]: Is this used somewhere?

Comment (tklauser) [reply test/helpers/cons.go:25]: I think this was automatically added by `goimports` (which my editor runs automatically on save). Can drop it if needed.

Comment (kkourt) [reply test/helpers/cons.go:25]: I see, thanks! Let's assume that go tooling knows what is doing and move on ;)

**Review by kkourt — APPROVED** [review]

**Pull request merged**